### ISSUE TITLE
Proposal for subdomain migration and management for web brand consolidation

### ISSUE THREAD

**Issue by ricetj** [department-of-veterans-affairs/va.gov-team#579]
## Intro
After a conversation with Patrick about the taking over a domain. There was an ask that @wyattwalter provide more insight in LOE.

Update: 
A proposal will be presented for how the team will take over a domain, on hold till July 26th.

## AC
Describe the follow:
- [x] LOE for taking over a domain: fixed cost + ongoing maintenance (overall and per-domain)
- [ ] Proposal for subdomain migration and management

**Comment by ricetj** [comment]
LOE doc was sent

**Comment by wyattwalter** [comment]
For now, we only have two sites migrating to the platform each taking a different approach (explore.va.gov, www.pittsburgh.va.gov). The plan is to compare once those are done and make a proposal going forward.

**Comment by ricetj** [comment]
Blocked by Pittsburgh takeover

**Comment by wyattwalter** [comment]
The Pittsburgh migration took a very different turn where subdomains are staying where they were and redirected outside of our control. Until this becomes a need again, I'm going to close this and get it out of our backlog.
